docs/cookbook/html-conversion.md

```diff
@@ -166,7 +166,9 @@ Each of the four pages included in this template has some common areas that can
 
 Inside the `views` folder create another folder named `fragments` and a file named `navigation.html`. To turn this page into a fragment, add opening and closing fragment block tags - `{% fragment navigationArea() %}` and `{% endfragment %}`.
 
-Open one of the template pages and copy the navigation section. Paste this between the two fragment tags. To add the website brand to the navigation, we will replace the `href` and logo in the first link with data from schema fields in the apostrophe global settings. Note that the homepage URL is populated from a `relationship` field. That data is delivered to the page in an array, so we need to specify that we are getting the `_url` from the first array item. Next, within the unordered list, delete the last three `<li>` items. To populate the list with each of the pages selected in the global settings we will use a `for` loop.
+Open one of the template pages and copy the navigation section. Paste this between the two fragment tags. To add the website brand to the navigation, we will replace the `href` with the homepage URL from `data.home`, which is available to all [templates](../guide/pages.md). We will add a simple text logo from user input in the apostrophe global settings.
```

Comment:
`data.home._url` might be a little more informative

Reply:
Changed.